sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/internal/state/AsynchronousMetricStorageTest.java

```diff
@@ -54,7 +58,7 @@
 
   @BeforeEach
   void setup() {
-    spyAttributesProcessor = Mockito.spy(AttributesProcessor.noop());
+    spyAttributesProcessor = spy(AttributesProcessor.noop());
```

Comment:
Since you happened to update this line (and maybe wish now you hadn't :-) ), consider migrating this to `@Spy`, generally prefer not to mix annotations and non-annotations in the same class without strong reason